src/compile/selection/selection.ts

```diff
@@ -239,17 +239,6 @@ export function channelSignalName(selCmpt: SelectionComponent, channel: Channel)
   return selCmpt.name + '_' + channel;
 }
 
-function clippedGroup(model: Model, marks: any[]): any[] {
-  return [{
-    type: 'group',
-    encode: {
-      enter: {
-        width: model.getSizeSignalRef('width'),
-        height: model.getSizeSignalRef('height'),
-        fill: {value: 'transparent'},
-        clip: {value: true}
-      }
-    },
-    marks: marks.map(model.correctDataNames)
-  }];
+function clipMarks(marks: any[]): any[] {
+  return marks.map((m) => (m.clip = true, m));
```

Comment:
We still need clipping, but with beta 29, we can do it directly on marks rather than nesting within a group. Good catch on the transparent fill. Force pushed an update.